layers/stp.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2017 Google, Inc. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file in the root of the source
+// tree.
+
+package layers
+
+import (
+	"github.com/google/gopacket"
+)
+
+// Spanning Tree Protocol is on top of LLC to transport BPDU (bridge protocol data unit) message.
+type STP struct {
+	BaseLayer
```

Comment:
I'm assuming the idea is to fill this in with actual STP fields at a later date.  Maybe add a TODO here?
```go
// TODO:  parse the STP protocol into actual subfields.
```

Reply:
done